### PR TITLE
Mark Internet#pass deprecated

### PR DESCRIPTION
**Pull request by snuyanzin**
`Internet#password` is very limited with its functionality and we should encourage users to use `Text` instead which covers much more cases

### CODE REVIEW

File: src/main/java/net/datafaker/providers/base/Internet.java [hunk 1]
```diff
@@ -176,26 +176,62 @@ public String httpMethod() {
         return resolve("internet.http_method");
     }
 
+    /**
+     * Consider usage of {@link net.datafaker.providers.base.Text#text} instead,
+     * look at net.datafaker.providers.base.TextTest#textShouldContain3RULowerCaseAnd5CustomSpecialSymbols()
```
Comment (kingthorin) [inline src/main/java/net/datafaker/providers/base/Internet.java:181]: RULower? Regular upper lower?

**Review by kingthorin — COMMENTED** [review]
Do we bundle unit tests in the library? It seems strange to document that users should check unit tests.

File: src/main/java/net/datafaker/providers/base/Internet.java (hunk 1 above)
Comment (snuyanzin) [reply src/main/java/net/datafaker/providers/base/Internet.java:181]: >textShouldContain3RULowerCaseAnd5CustomSpecialSymbols

means that text should contain 3 lower case letters from `RU` locale and 5 custom special symbols

**Comment by snuyanzin** [comment]
>Do we bundle unit tests in the library?

no
> It seems strange to document that users should check unit tests.

yep, there are 2 ways I see: either put example in javadoc for `net.datafaker.providers.base.Text#text` or in site documentation

**Comment by snuyanzin** [comment]
Ok, added to javadoc

**Review by kingthorin — APPROVED** [review]
Thanks, I think that's much more usable

**Comment by bodiam** [comment]
I've reverted this change. Without these methods it's way to complex too create a password, and I don't see why these methods can't exists next to each other.

**Pull request merged**